### PR TITLE
Akoines/react devise

### PR DESCRIPTION
**Pull request by akoinesjr**
Installs Bootstrap, additional and modifications to styling, sets up interface for users to sign up / in / out.

### CODE REVIEW

File: app/serializers/user_serializer.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@
 #  account_id             :integer          not null
 #
 
-class User < ActiveModel::Serializer
+class UserSerializer < ActiveModel::Serializer
```
Comment (pboling) [inline app/serializers/user_serializer.rb:28]: Gah, so many typos in my rush-code.  Thanks for fixing!

**Review by pboling — APPROVED** [review]
Looks great!

File: app/serializers/user_serializer.rb (hunk 1 above)
Comment (akoinesjr) [reply app/serializers/user_serializer.rb:28]: Actually I think you uncovered / fixed this elsewhere as we were rushing to deploy during the Hackathon and it just never got merged.

Comment (pboling) [reply app/serializers/user_serializer.rb:28]: Ah, that's why there are some commits from me in this PR. :D

**Pull request merged**